unix_commands/ls.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/python3
+
+# Importing packages
+import argparse
+import os
+
+# Parsing arguments
+from argparse import ArgumentParser
+
+parser = ArgumentParser(description="ls analogue")
+parser.add_argument('-a', '--all', action='store_true', help="Do not hide files starting with .")
+parser.add_argument('path', metavar = 'PATH', nargs='?', default='./')
+args = parser.parse_args()
+
+my_path = args.path
+
+files = os.listdir(my_path)
+files_no_dot = []
+
+if args.all:
+    print(' '.join(files))
```

Comment:
```suggestion
    print('\n'.join(files))
```
Лучше так, потом это будет проще отфильтровывать и вообще работать с выводом (если передаём дальше по конвейеру)